Knoema.Client/Data/FullDimensionRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Knoema.Data
+{
+	public class FullDimensionRequest
```

Comment:
Check that your spacing settings matches the ones in the project. Looks like you're using spaces instead of tabs.

Reply:
I used tabs